addons/website/static/src/snippets/s_website_form/options.js

```diff
@@ -1205,6 +1207,16 @@ options.registry.WebsiteFieldEditor = FieldEditor.extend({
      * @override
      */
     _renderCustomXML: async function (uiFragment) {
+        // In case the file visibility selector does not exist, create it.
+        if (!uiFragment.querySelector('we-select[data-name="hidden_condition_file_opt"]')){
+            uiFragment.querySelector('we-row[data-name="hidden_condition_opt"]').append($(`
+                <we-select data-name="hidden_condition_file_opt" data-attribute-name="visibilityComparator" data-no-preview="true">
+                    <we-button data-select-data-attribute="fileSet">` + _t("Is set") + `</we-button>
+                    <we-button data-select-data-attribute="!fileSet">` + _t("Is not set") + `</we-button>
```

Comment:
Instead of summing template literals, you can just do this (that's often the main point when using template literals instead of strings):
```javascript
`
<blabla>${_t("Is set")}</blabla>
`
```

Reply:
Yes - haha you're speaking to a JS noob who forgot about that! Sorry!